### PR TITLE
Add link to gem page in stats

### PR DESCRIPTION
**Pull request by nicolasleger**

### CODE REVIEW

File: app/views/stats/index.html.erb [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 <% @most_downloaded.each do |gem| %>
   <div class="stats__graph__gem">
-    <h3 class="stats__graph__gem__name"><%= gem.name %></h3>
+    <h3 class="stats__graph__gem__name"><a class="gems__gem" href="<%= rubygem_path(gem) %>"><%= gem.name %></a></h3>
```
Comment (arthurnn) [inline app/views/stats/index.html.erb]: Maybe use the rails link helper here ?

Comment (nicolasleger) [inline app/views/stats/index.html.erb]: I thought too but it's not already used in `_rubygem.html.erb` where I took the link syntax.
Agree to use `link_to` in both?

Comment (arthurnn) [inline app/views/stats/index.html.erb]: :+1:

**Comment by arthurnn** [comment]
thanks

**Pull request merged**